source/blueprints/overview.rst

```diff
@@ -94,7 +106,31 @@ Build the Natural Language Processing models that power the app.
 3. Run
 ^^^^^^
 
-Interact with the app in the Python shell using the commands below. Try the queries shown in the examples, then try some queries of your own invention.
```

Comment:
Interact with the app using the commands below. Try the queries shown in the examples, then try some queries of your own invention.